### ISSUE TITLE
Archive feedstock?

### ISSUE THREAD

**Issue by jakirkham** [conda-forge/netifaces-feedstock#35]
Looks like [the upstream repo]( https://github.com/al45tair/netifaces ) has been archived

![Image](https://github.com/user-attachments/assets/75e0ee81-6b95-4f89-b20e-848bc93bdd39)

Given this, should this feedstock also be archived?

**Comment by minrk** [comment]
Makes sense. Most things I've seen are moving to psutil's equivalent functionality
